### DISCUSSION TITLE
Tibber und HM

### DISCUSSION THREAD

**Discussion by Gamehunter1989** [OpenCCU/OpenCCU#2392] [🇩🇪 Generelle Diskussionen]
Hallo zusammen,

bin auf Tibber aufmerksam geworden. 
Wer Tibber nicht kennt 
Kurz zusammengefasst; Tibber ist ein Ököstromanbieter welcher eine stündliche Abrechnung zum Börsentarif anbietet.

Tibber hat eine API Schnittstelle, diese würde ich gerne nutzen, um meine Zählerdaten an Tibber zu senden. Tibber hat einen Sensor, welcher die Daten des Zählers ausließt, wie auch der, der HM. Da ich diesen Sensor von EQ3 verwende, bin ich eigentlich nicht bereit 90 € für einen Sensor auszugeben, hab ja einen.

Hat jemand hier Erfahrungen mit Tibber und meinem Anliegen?

Freue mich auch Feedback.

LG Mark

**Comment by Baxxy13** [comment]
Die Frage wäre im Homematic-Forum besser aufgehoben. Da tummeln sich mehr User die von sowas Ahnung haben.

Aber grundsätzlich:
Wenn Tibber einen "selbst abgelesenen" Zählerstand (geht ja nur mit der IEC-Version des HM-ES-TX-WM), den du per API übermittelst, akzeptiert dann sollte das machbar sein. Kommt aber auch drauf an wie da die Authentifizierung umgesetzt ist, nicht alles lässt sich mit RaspberryMatic - Bordmitteln umsetzen.

**Comment by jens-maus** [comment]
Zusätzlich würde ich aber anmerken wollen das IMHO solche Dinge besser in einer potenteren Logikengine bzw. Middleware als der ReGaHss in RaspberryMatic/CCU3 umgesetzt werden sollte. Für HomeAssistant gibt es z.B. das hier: https://www.home-assistant.io/integrations/tibber/ und für ioBroker: https://github.com/Codibris/ioBroker.tibberconnect. Kurzum: Wenn man an einer umfassenden Hausautomation interessiert ist die Dinge aus mehreren Welten vereinen kann (eben auch solche externen Dienste) kommt man heutzutage um HomeAssistant oder ioBroker IMHO nicht drumherum..

**Comment by Gamehunter1989** [comment]
Sehr gute Hinweise, iobroker hatte ich auch mal in seiner Anfangszeit verwendet, der wird mittlerweile sehr mächtig sein. Werde ich mir mal anschauen. 

Die Frage ist ob Tibber die Werte (egal wie die ankommen)überhaupt auswerten/verwenden möchte.
